Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -3968,9 +3968,6 @@ void Element::play_or_cancel_animations_after_display_property_change()
     auto play_or_cancel_depending_on_display = [&](Animations::Animation& animation) {
         if (has_display_none_inclusive_ancestor) {
             animation.cancel();
-        } else {
-            HTML::TemporaryExecutionContext context(realm());
-            animation.play().release_value_but_fixme_should_propagate_errors();
```

Comment:
I think we do want to do this, but only when the display property actually gets changed from `none` to something else. Currently `play_or_cancel_animations_after_display_property_change()` gets called in `Element::inserted()` and `Element::removed()` and this branch will be taken in those cases when it shouldn't be.

If this branch definitely isn't supposed to be there then the method name should probably be changed.

Reply:
I've verified the behaviour locally, do you want me to add tests?